client/components/utils/add-comment-plugin.js

```diff
@@ -78,22 +78,20 @@ export default function addCommentPlugin(vueInstance) {
 }
 
 export function addHighlight(keys, schema, state, from, to, dispatch) {
-  const {doc, tr} = state;
-  let attrs = null;
-  if (!doc.rangeHasMark(from, to, schema.marks.highlight)) {
-    attrs = {"highlight-keys": keys};
-    if (!attrs["highlight-keys"]) {
-      return false;
-    }
+  const {tr: transaction} = state;
+  const attrs = {"highlight-keys": keys};
+  if (!attrs["highlight-keys"]) {
```

Comment:
I am still not getting this. Aren't you still then checking here if `keys` is falsy value? When would that be/happen?